src/scripts/OSFramework/OSUI/Pattern/Progress/Circle/scss/_progresscircle.scss

```diff
@@ -77,7 +77,7 @@
 		z-index: var(--layer-local-tier-1);
 	}
 
-	.svg-wrapper {
+	.osui-inline-svg {
```

Comment:
@BenOsodrac For instance on the `osui-progress-circle` you kept svg-wrapper. Shouldn't we have both `svg-wrapper` and `osui-inline-svg`?

Reply:
Don't think so, on the CSS we should use the most recent classes, no need to keep referencing old ones. The OutSystems UI is released as a bulk, so this CSS will have correct structure on the OutSystems UI module' html.